resources/shared/params.mjs

```diff
@@ -146,21 +146,25 @@ class Params {
         return shuffleSeed;
     }
 
-    toSearchParams() {
+    toSearchParams(forRemote = false) {
         const rawParams = { ...this };
         rawParams["viewport"] = `${this.viewport.width}x${this.viewport.height}`;
+
+        if (forRemote) {
+            delete rawParams["suites"];
+            delete rawParams["tags"];
+        }
         return new URLSearchParams(rawParams).toString();
     }
 }
 
 export const defaultParams = new Params();
 
-const searchParams = new URLSearchParams(window.location.search);
+const searchParams = new URLSearchParams(typeof window !== "undefined" ? window.location.search : undefined);
```

Comment:
when is it happening that window isn't present?

optional nit: use `?.` operator